src/src/com/microsoft/aad/adal/CacheKey.java

```diff
@@ -155,6 +155,14 @@ public static String createMultiResourceRefreshTokenKey(AuthenticationRequest it
         return createCacheKey(item.getAuthority(), item.getResource(), item.getClientId(), true,
                 cacheUserId);
     }
+    
+    public static String createFamilyRefreshTokenKey(final AuthenticationRequest authRequest, final String userId) {
+        if (authRequest == null) {
+            throw new IllegalArgumentException("authentication request is null");
+        }
+        
+        return createCacheKey(authRequest.getAuthority(), null, null, true, userId);
```

Comment:
I would suggest to keep family id somewhere in cache. That way you don't need to hardcode familyId. 
And it would automatically work for another familyId